main.go

```diff
@@ -43,6 +43,7 @@ const (
 	rateLimiterFrequencyDefault = 30
 	failureBaseDelayDefault     = 1 * time.Second
 	failureMaxDelayDefault      = 1000 * time.Second
+	retryTimeDefault            = 10 * time.Hour
```

Comment:
Should we call this `requeueAfterTime`, since retry indicates that something failed and I need to try it again, but this is just a time that will also be used for non-error scenarios. Another idea is `reconciliationInterval`?